efel/pyfeatures/pyfeatures.py

```diff
@@ -140,6 +141,45 @@ def strict_burst_number() -> np.ndarray:
     return np.array([burst_mean_freq.size])
 
 
+def all_burst_number():
```

Comment:
Wouldn't it be more descriptive to call it something like  `count_burst_events_by_isi_clustering`?